.env

```diff
@@ -42,10 +43,23 @@ OKP4_GITHUB_PREDICATE_ADDRESS_URL='https://github.com/okp4/okp4d/blob/v4.1.0/x/l
 OKP4_GITHUB_LOGIC_MODULE_PREDICATES_URL='https://github.com/okp4/okp4d/tree/v4.1.0/x/logic/predicate'
 OKP4_GITHUB_PROLOG_TEMPLATE_URL='https://github.com/okp4/contracts/tree/v2.0.0/contracts/okp4-law-stone/examples/multiple-sources'
 OKP4_GITHUB_ONTOLOGY_RHIZOME_URL='https://github.com/okp4/ontology/blob/main/example/rhizome/dataset/0ea1fc7a-dd97-4adc-a10e-169c6597bcde.ttl'
+OKP4_GITHUB_ONTOLOGY_S3_SERVICE_URL='https://github.com/okp4/ontology/blob/main/src/core.ttl#L109'
+OKP4_GITHUB_S3_SERVICE_METADATA_URL='https://github.com/okp4/ontology/blob/main/src/metadata-service-general.ttl'
+OKP4_GITHUB_ONTOLOGY_CSV_DATASET_URL='https://github.com/okp4/ontology/blob/main/src/core.ttl#L44'
+OKP4_GITHUB_ONTOLOGY_CSV_DATASET_METADATA_URL='https://github.com/okp4/ontology/blob/main/src/metadata-dataset-general.ttl'
+OKP4_GITHUB_SERVICE_REFERENCE_URL='https://github.com/okp4/ontology/blob/main/example/rhizome/service/d1b0b4d3-f9a6-4115-bcd8-ad97233a7b08.ttl'
+OKP4_GITHUB_DATASET_REFERENCE_URL='https://github.com/okp4/ontology/blob/main/example/rhizome/dataset/1a4c5896-b13e-40f7-a603-5a5b3ed996a1.ttl'
+OKP4_GITHUB_COGNITARIUM_PROLOG_CODE_URL='https://github.com/okp4/contracts/blob/main/contracts/okp4-cognitarium/examples/prolog-query/query.pl'
+OKP4_GITHUB_COGNITARIUM_SERVICE_CATEGORY_URL='https://github.com/okp4/ontology/blob/main/src/vocab-category-service.ttl'
+OKP4_GITHUB_COGNITARIUM_DATASET_MEDIATYPE_URL='https://github.com/okp4/ontology/blob/main/src/vocab-mediatype.ttl'
+OKP4_GITHUB_STORAGE_SERVICE_URL='https://github.com/okp4/ontology/blob/main/example/rhizome/service/d1b0b4d3-f9a6-4115-bcd8-ad97233a7b08.ttl#L10'
+OKP4_GITHUB_DATASET_URL='https://github.com/okp4/ontology/blob/main/example/rhizome/dataset/1a4c5896-b13e-40f7-a603-5a5b3ed996a1.ttl#L21C10-L21C10'
```

Comment:
The links are not correct. Use persistent links based on Git hashes or tags to ensure the stability and prevent broken or outdated references. It's especially important when you're designating a line in a source file that's likely to change the next time you make a change.

Reply:
I updated the links [here](https://github.com/okp4/dev-kanban/issues/1551), but too late. 
Should we provide persistent links for each GitHub link, or only for those with line anchors?

Reply:
Links proposal: 

```
OKP4_GITHUB_ONTOLOGY_S3_SERVICE_URL='https://github.com/okp4/ontology/blob/main/src/core.ttl'
OKP4_GITHUB_S3_SERVICE_METADATA_URL='https://github.com/okp4/ontology/blob/main/src/metadata-service-general.ttl'
OKP4_GITHUB_ONTOLOGY_CSV_DATASET_URL='https://github.com/okp4/ontology/blob/main/src/core.ttl'
OKP4_GITHUB_ONTOLOGY_CSV_DATASET_METADATA_URL='https://github.com/okp4/ontology/blob/main/src/metadata-dataset-general.ttl'
OKP4_GITHUB_SERVICE_REFERENCE_URL='https://github.com/okp4/ontology/blob/main/example/rhizome/service/d1b0b4d3-f9a6-4115-bcd8-ad97233a7b08.ttl'
OKP4_GITHUB_DATASET_REFERENCE_URL='https://github.com/okp4/ontology/blob/main/example/rhizome/dataset/1a4c5896-b13e-40f7-a603-5a5b3ed996a1.ttl'
OKP4_GITHUB_COGNITARIUM_PROLOG_CODE_URL='https://github.com/okp4/contracts/blob/main/contracts/okp4-cognitarium/examples/prolog-query/query.pl'
OKP4_GITHUB_COGNITARIUM_SERVICE_CATEGORY_URL='https://github.com/okp4/ontology/blob/main/src/vocab-category-service.ttl'
OKP4_GITHUB_COGNITARIUM_DATASET_MEDIATYPE_URL='https://github.com/okp4/ontology/blob/main/src/vocab-mediatype.ttl'
OKP4_GITHUB_STORAGE_SERVICE_URL='https://github.com/okp4/ontology/blob/d3209eb9395a0292927627004e5114b63bf3cab0/example/rhizome/service/d1b0b4d3-f9a6-4115-bcd8-ad97233a7b08.ttl#L10'
OKP4_GITHUB_DATASET_URL='https://github.com/okp4/ontology/blob/d3209eb9395a0292927627004e5114b63bf3cab0/example/rhizome/dataset/1a4c5896-b13e-40f7-a603-5a5b3ed996a1.ttl#L21C20-L21C39'
```